server/config/cors.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Laravel CORS
+    |--------------------------------------------------------------------------
+    |
+    | allowedOrigins, allowedHeaders and allowedMethods can be set to array('*')
+    | to accept any value.
+    |
+    */
+
+    'supportsCredentials' => false,
+    'allowedOrigins' => ["http://localhost:3000", "http://localhost:3001"],
```

Comment:
use .env to define origins

Reply:
Done.
Commit: https://github.com/Levayv/ipma_v2/commit/699e165bf482b16f63cbe726aba48ba47cd124e7#diff-d8334e019cb77a3cd0bbae0476d64db1R42